.ci/azure-steps.yml

```diff
@@ -0,0 +1,42 @@
+steps:
+
+- script: |
+    call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" $(python.architecture)
+    set MSSdk=1
+    set DISTUTILS_USE_SDK=1
+    python -m pip wheel . -w wheelhouse/
+  displayName: 'Build wheel (Windows Python 2.7)'
+  condition: and(succeeded(), eq(variables['Agent.OS'], 'Windows_NT'), eq(variables['python.version'], '2.7')) 
+
+- script: |
+    python -m pip wheel . -w wheelhouse/
+  displayName: 'Build wheel'
+  condition: and(succeeded(), not(and(eq(variables['Agent.OS'], 'Windows_NT'), eq(variables['python.version'], '2.7'))))
+
+- script: |
+    ls -lh wheelhouse
+    mkdir -p dist
+    cp wheelhouse/$(package_name)* dist/.
+  displayName: 'Show wheelhouse'
+
+- script: |
+    python -m pip install $(package_name) --no-index -f wheelhouse
+  displayName: 'Install wheel'
+
+- script: |
+    python -m pytest --pyargs iminuit --junitxml=junit/test-results.xml
```

Comment:
FYI, this is the block with the workaround that is not consistent with a general script for all packages. `workingDirectory` and `--pyargs iminuit`.